File: day-01/solution/main.cpp
#include <iostream>

auto main() -> int {

    int a{};
    std::cin >> a;

    int b{};
    std::cin >> b;

    for (auto i{a}; i <= b; i++) {
       if (i % 3 == 0) std::cout << "Foo\n";
       else if (i % 2 == 0) std::cout << "Bar\n";
       else std::cout << "Baz\n";
    }
    
}

File: day-01/solution/Main.kt
fun main() {
    val a = readln().toInt()
    val b = readln().toInt()

    (a..b).forEach {
        when {
            it % 3 == 0 -> println("Foo")
            it % 2 == 0 -> println("Bar")
            else -> println("Baz")
        }
    }
}
